app/views/events/_form.html.haml

```diff
@@ -1,7 +1,9 @@
 = simple_form_for resource do |f|
   .section
-    = render 'form_base', f: f
```

Comment:
Oh, phantastic. Didn't know abt ```&.````. This is awesome stuff. 👍
And I could have used the ```f```-var, somehow this was a leftover from me mangling things with byebug or so...
Thx for the lessons :)